security/tor/src/opnsense/mvc/app/controllers/OPNsense/Tor/Api/ExitaclController.php

```diff
@@ -41,129 +39,26 @@ class ExitaclController extends ApiMutableModelControllerBase
     static protected $internalModelClass = '\OPNsense\Tor\ACLExitPolicy';
     public function searchaclAction()
     {
-        $this->sessionClose();
```

Comment:
I don't want them here - everything should be handled in the base* functions in core where they seem to be missing at the moment (no redundancy in the controllers).

@fichtner can you amend that in core?

Reply:
Session close is not a boilerplate decision to be pushed to the mutable controllers for data manipulation... because we manipulate data and don’t want concurrent access... maybe these can go, but they shouldn’t be in base st all.

Reply:
ok - so I will add that tomorrow.